playbooks/acs.yml

```diff
@@ -51,6 +51,11 @@
   roles:
     - role: '../roles/search'
   post_tasks:
+    - name: Make sure Solr is running
+      service:
+        name: alfresco-search
+        status: started
```

Comment:
is this a new pattern we want to adhere with?
(ensure service are running only when running the playbook and not into the role tasks?)

maybe we want also to wait for solr really ready